src/C4Config.cpp

```diff
@@ -578,9 +625,21 @@ void C4ConfigGeneral::DeterminePaths(bool forceWorkingDirectory)
 
 char AtPathFilename[_MAX_PATH + 1];
 
-const char *C4Config::AtExePath(const char *szFilename)
+const char *C4Config::AtExePath(const char *szFilename, bool forceExeFolder)
```

Comment:
Umm, no. That function explicitely returns a path to a file inside the executable directory. It must *never* return something else, otherwise we could throw it out of the window. This isn't the right place to implement the required functionality.